Examples/Modules/laser_injection/inputs_2d_rt

```diff
@@ -31,19 +31,19 @@ warpx.cfl = 1.0
 # Laser
 lasers.names        = laser1
 laser1.profile      = Gaussian
-laser1.position     = 9.e-6  0.  0.  # This point is on the laser plane
-laser1.direction    = 1.     0.  0.  # The plane normal direction
-laser1.polarization = 0.     1.  0.  # The main polarization vector
+laser1.position     = 10.e-6  0.e-6  0.e-6  # This point is on the laser plane
+laser1.direction    = 2.  0.  1.  # The plane normal direction
+laser1.polarization = 1. 1.  -2.  # The main polarization vector
 laser1.e_max        = 4.e12          # Maximum amplitude of the laser field (in V/m)
-laser1.wavelength   = 0.8e-6         # The wavelength of the laser (in meters)
+laser1.wavelength   = 1.0e-6         # The wavelength of the laser (in meters)
 laser1.profile_waist = 5.e-6       # The waist of the laser (in meters)
-laser1.profile_duration = 15.e-15  # The duration of the laser (in seconds)
-laser1.profile_t_peak = 30.e-15    # The time at which the laser reaches its peak (in seconds)
-laser1.profile_focal_distance = 100.e-6  # Focal distance from the antenna (in meters)
+laser1.profile_duration = 10.e-15  # The duration of the laser (in seconds)
+laser1.profile_t_peak = 20.e-15    # The time at which the laser reaches its peak (in seconds)
+laser1.profile_focal_distance = 13.109e-6  # Focal distance from the antenna (in meters)
```

Comment:
There isn't a very strong reason. I chose that value so that the laser is at focus at the end of the simulation. I have added a short comment to explain that.